app/controllers/games_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class GamesController < ApplicationController
-  before_action :authenticate_user!, only: %i[new create show update_invited_user surrender]
+  before_action :authenticate_user!, only: %i[new create show update_invited_user surrender draw]
+  skip_before_action :verify_authenticity_token, only: %i[draw]
```

Comment:
I was getting the following error in Rspec
` ActionController::InvalidCrossOriginRequest:
       Security warning: an embedded <script> tag on another site requested protected JavaScript. If you know what you're doing, go ahead and disable forgery protection on this action to permit cross-origin JavaScript embedding.`
Line 5 solved this problem. If anyone knows another solution that would be great.

Reply:
I think this might be related to the JS click event you have in `show.html.erb`. See my comment on that one.